cmd/kube-controller-manager/app/options/options.go

```diff
@@ -163,5 +164,6 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&s.KubeAPIBurst, "kube-api-burst", s.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
 	fs.DurationVar(&s.ControllerStartInterval.Duration, "controller-start-interval", s.ControllerStartInterval.Duration, "Interval between starting controller managers.")
 	fs.BoolVar(&s.EnableGarbageCollector, "enable-garbage-collector", s.EnableGarbageCollector, "Enables the generic garbage collector. MUST be synced with the corresponding flag of the kube-apiserver. WARNING: the generic garbage collector is an alpha feature.")
+	fs.Int32Var(&s.ConcurrentGCSyncs, "concurrent-gc-syncs", s.ConcurrentGCSyncs, "The number of garbage collectors that are allowed to sync concurrently. Larger number = more responsive garbage collector, but more CPU (and network) load")
```

Comment:
@249043822 Because the GC has more components than the workers, so could you reword the sentence to "The number of running garbage collector workers. Larger number makes the garbage collector more responsive, but also leads to heavier CPU and network load"?